ouroboros-network/src/Ouroboros/Network/Diffusion/P2P.hs

```diff
@@ -825,6 +830,7 @@ runM Interfaces
               makeConnectionHandler
                 dtLocalMuxTracer
                 SingResponderMode
+                noBindForkPolicy
```

Comment:
I think that local sockets should use `daMuxForkPolicy` too.
Historically `cardano-cli` has been able to but a lot of load on a `cardano-node.

Reply:
Done.